en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md

```diff
@@ -155,30 +160,30 @@ This tool is packaged with WSO2 API Manager by default. Follow the steps below t
     <td>The pseudonym with which the username should be replaced.<html><div class="admonition note"><p class="admonition-title">Note</p><p> Let's say you run the tool to replace all references to a particular deleted user’s identity, and then you add the same user back to the system for some reason, and later you want to delete the user again and replace all references to the user with the same pseudonym that was used the first time. To do this you need to specify a pseudonym when you run the tool the first time and ensure that you use that same pseudonym when you run the tool the second time. </p></div></html></td>
     <td>No</td>
     <td>A random UUID value is generated as the pseudonym.</td>
-    <td><p><code>                -pu “123-343-435-545-dfd-4”               </code></p></td>
+    <td><p><code>-pu “123-343-435-545-dfd-4”</code></p></td>
     </tr>
     <tr class="odd">
     <td>carbon</td>
     <td><p>The CARBON HOME.</p>
     <br />
 
-    <p>This should be replaced with the variable <code>                $CARBON_HOM               </code> E in directories configured in the main configuration file.</p></td>
+    <p>This should be replaced with the variable <code>$CARBON_HOME</code> in directories configured in the main configuration file.</p></td>
     <td>No</td>
     <td><br />
     </td>
-    <td><code>               -carbon “usr/bin/wso2am/wso2am3.2.0”              </code></td>
+    <td><code>-carbon “usr/bin/wso2am/wso2am3.2.0”</code></td>
     </tr>
     </tbody>
     </table>
 
 4.  All references to the user are removed from WSO2 API Manager. You can view the generated reports inside the `<API-M_HOME>/repository/components/tools/forget-me/conf` directory. Reports will be generated with the naming convention of `Report-<PROCESSOR>-<TIMESTAMP>.txt`  in your current working directory.
-    eg: Report-log-file-1598483873677.txt
+    Example:  `Report-log-file-1598483873677.txt`
 
-#### Running the toolkit in standalone mode
+### Running the toolkit in standalone mode
 
 This tool can run standalone and therefore cater to multiple products. This means that if you are using multiple WSO2 products and need to delete the user's identity from all products at once, you can do so by running the tool in standalone mode.
 For information on how to build and run the Forget-Me tool, see [Removing References to Deleted User Identities in WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Removing+References+to+Deleted+User+Identities+in+WSO2+Products) in the WSO2 Administration Guide.
```

Comment:
This link does not seem to be applicable anymore as we are not supposed to point to the Admin guide. The content of that page seems to have been added to this page.

Reply:
That doc speaks about deleting user entries from multiple wso2 products. So we need to use https://github.com/wso2/identity-anonymization-tool for that purpose and that doc specifies it.